### PR TITLE
Replace calls to CFStringCreateWithCString with constant CFStrings

### PR DESCRIPTION
**Pull request by AtariDreams**
This would save performance by removing unnecessary calls to CFStringCreateWithCString

### CODE REVIEW

**Comment by AtariDreams** [comment]
@compnerd Is this good?

**Comment by compnerd** [comment]
CC: @parkera

File: CoreFoundation/Base.subproj/CFPlatform.c [hunk 1]
```diff
@@ -276,8 +276,11 @@ CF_PRIVATE CFStringRef _CFProcessNameString(void) {
     static CFStringRef __CFProcessNameString = NULL;
     if (!__CFProcessNameString) {
         const char *processName = *_CFGetProgname();
-        if (!processName) processName = "";
-        CFStringRef newStr = CFStringCreateWithCString(kCFAllocatorSystemDefault, processName, kCFPlatformInterfaceStringEncoding);
+        CFStringRef newStr;
+        if (processName)
```
Comment (parkera) [inline CoreFoundation/Base.subproj/CFPlatform.c:280]: Our code style is to always use `{ }` on multi-line conditionals.

**Review by parkera — CHANGES_REQUESTED** [review]
I think using constant strings here is fine. One code style fix that needs to be made first before merging.

**Comment by AtariDreams** [comment]
@parkera Done!

**Comment by parkera** [comment]
@swift-ci test

**Comment by AtariDreams** [comment]
I apologize. This has been fixed. Let's try again @parkera

**Comment by AtariDreams** [comment]
@compnerd Could you please run the tests again and merge

**Comment by compnerd** [comment]
@swift-ci please test

**Comment by AtariDreams** [comment]
macOS build seems to be broken; it isn't even starting. Huh.

**Comment by AtariDreams** [comment]
@compnerd Seems like the macOS Jenkins instance is broken; it will not start the pipeline

**Comment by AtariDreams** [comment]
Okay everything is fixed! @compnerd Let's try it one more time. It builds.

**Comment by AtariDreams** [comment]
@swift-ci test

**Comment by AtariDreams** [comment]
@parkera Can we give this one more go as well?

**Comment by parkera** [comment]
@swift-ci test

**Comment by AtariDreams** [comment]
@parkera Everything is fixed. Can we do this one more time?

**Comment by AtariDreams** [comment]
@compnerd Could you please run the tests again and merge?

**Comment by AtariDreams** [comment]
Can someone please do the please test and merge thing? @compnerd @parkera

**Comment by parkera** [comment]
@swift-ci test

**Comment by AtariDreams** [comment]
> @swift-ci test

It passed! Can we merge please? @parkera

**Comment by compnerd** [comment]
This might be regressing the Windows build:
https://ci-external.swift.org/job/oss-swift-windows-toolchain-x86_64-vs2019/2845/console

**Pull request merged**